lib/tasks/attempts.rake

```diff
@@ -78,8 +77,66 @@ namespace :attempts do
     puts 'Remember to restart Rails after updating application.yml.default!' if failed
   end
 
-  desc 'Clear all events from Redis'
-  task purge_events: :environment do
-    IrsAttemptsApi::RedisClient.clear_attempts!
+  desc 'Generate a simple gzipped file'
+  task write_event: :environment do
+    # message = "Testing gzip compression. If you can read this, it works!"
+    #
+    # file = File.new('./gzip-test.txt.gz', 'wb')
+    # file.write Zlib.gzip(message)
+    # file.close
+
+    events = [_generate_event, _generate_event]
+
+    decoded_key_der = Base64.strict_decode64(IdentityConfig.store.irs_attempt_api_public_key)
+    pub_key = OpenSSL::PKey::RSA.new(decoded_key_der)
+
+    result = IrsAttemptsApi::EnvelopeEncryptor.encrypt(
+      data: events.join("\r\n"),
+      timestamp: Time.zone.now,
+      public_key: pub_key,
+    )
+
+    file = File.open(result.filename, 'wb')
+    file.write(result.encrypted_data)
+    puts "IV: #{Base64.encode64(result.iv)}"
+    puts "Encrypted key: #{Base64.encode64(result.encrypted_key).gsub("\n", '')}"
+  end
+
+  task decrypt_file: :environment do
+    iv = Base64.strict_decode64(ENV['IRS_IV'])
+    encrypted_key = Base64.strict_decode64(ENV['IRS_KEY'])
+    puts "File is #{ARGV[1]}"
```

Comment:
This isn't at all user-friendly, but it's temporary.

Running `rake attempts:write_event` will generate a file (find it with `ls -althr` -- I should just print the path out above) with 2 events in it, and spit out the IV and encrypted key.

You should be able to do something like:
```
# both values from the output in write_events
export IRS_IV="..."
export IRS_KEY="..."
be rake attempts:decrypt_file FILENAME_FROM_ABOVE
```

Fascinatingly, this is choking on the gunzip step in `EnvelopeEncryptor.decrypt`, which I otherwise could not reproduce. This seems to match what the IRS is seeing.